src/client/index.html

```diff
@@ -4,19 +4,14 @@
     <meta charset="UTF-8" />
     <title>Fit Together</title>
     
-    <!-- Bootstrap CSS -->
     <link href="assets/lib/bootstrap.min.css" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/css/materialize.min.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    
-	  <!-- jQuery -->
+
     <script src="assets/lib/jquery-2.2.4.min.js"></script>
-    
-    <!-- Bootstrap JS -->
     <script src="assets/lib/bootstrap.min.js"></script>
     
-	  <!-- Custom CSS -->
```

Comment:
why are we removing comments? dont these make sense to improve the readability of our code?